pages/account/manage/event/[[...data]].js

```diff
@@ -336,11 +336,11 @@ export default function ManageEvent({ BASE_URL, event }) {
 
                 <div className="mt-6 flex items-center justify-end gap-x-6">
                   {event._id && (
-                    <Button type="button" onClick={() => setOpen(true)}>
+                    <Button type="button" onClick={() => setOpen(true)} style={{marginBottom: '1.5rem'}}>
                       DELETE
                     </Button>
                   )}
-                  <Button type="submit" primary={true}>
+                  <Button type="submit" primary={true} style={{marginBottom: '1.5rem'}}>
```

Comment:
```suggestion
                  <Button type="submit" primary={true} className='mb-6'>
```

Reply:
I tried adding mb-6 to the classname as you have suggested but that didn't work, so I used style element

**Using className='mb-6'**

![image](https://github.com/EddieHubCommunity/BioDrop/assets/103899034/9e9ec678-7080-4c4e-a2eb-739b63505816)

![image](https://github.com/EddieHubCommunity/BioDrop/assets/103899034/a0a6f643-9251-4cff-a68f-6f8f192da8dc)

Reply:
let me get this straight, my change causes the save button to also lose its styling of colour etc

Reply:
I have successfully implemented by using Tailwind classes